docs/articles/getting-started.md

```diff
@@ -1,37 +1,37 @@
----
-uid: getting-started
----
-
```

Comment:
This is used for the internal IDs in DocFX, shouldn't be removed. (This goes for all other similar [front matter](https://jekyllrb.com/docs/front-matter/) in the docs as well.)